src/main/java/org/dependencytrack/policy/cel/mapping/ProjectProjection.java

```diff
@@ -41,8 +41,8 @@ public class ProjectProjection {
     @MappedField(sqlColumnName = "CLASSIFIER")
     public String classifier;
 
-    @MappedField(protoFieldName = "is_active", sqlColumnName = "ACTIVE")
-    public Boolean isActive;
+    @MappedField(protoFieldName = "inactive_since", sqlColumnName = "INACTIVE_SINCE")
+    public Date inactiveSince;
```

Comment:
Hmmm, just checked - it seems this entire class is no longer used. It's referenced in `CelPolicyQueryManager#fetchProject`, but that method is unused. Instead of adjusting this line, you could delete said code instead.

Reply:
It is also being used by `CelPolicyDao#loadRequiredFields` to filter sql columns.

Reply:
You're right! My bad!

Reply:
I'll delete this field from projection though.